yb-voyager/cmd/import.go

```diff
@@ -34,12 +34,13 @@ var importCmd = &cobra.Command{
 	Long:  ``,
 
 	PreRun: func(cmd *cobra.Command, args []string) {
-		validateImportFlags(cmd)
+		validateImportFlags()
+		sourceDBType = ExtractMetaInfo(exportDir).SourceDBType
```

Comment:
This will cause an issue with `import data file`!

Reply:
how? we are not calling some Parent PreRun in `importDataFileCommand`

Reply:
Ah! my bad! I thought it is a `PersistentPreRun`.